src/widgets/example.js

```diff
@@ -17,11 +17,11 @@ export default async function example(scopes) {
   const res = await ActivityReport.findAll({
     attributes: [
       [sequelize.fn('COUNT', sequelize.fn('DISTINCT', sequelize.col('"ActivityReport".id'))), 'numReports'],
-      // I really don't like `"activityRecipients->grant->grantee"."id"` but couldn't find a way
```

Comment:
no particular reason to do it in this PR, but do you think we could delete this example widget files, maybe as a tech debt ticket? here, and on the frontend if they still exist there. that way we don't have to continuously upkeep them

Reply:
Yeah I'll add a ticket

Reply:
https://ocio-jira.acf.hhs.gov/browse/TTAHUB-537